<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Helicopter_CANWindow</class>
 <widget class="QWidget" name="Helicopter_CANWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="CAN_GB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1401</width>
     <height>741</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QListWidget" name="lst_VID">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>391</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_VID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VID</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_MAB">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>320</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_MAB">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MAB</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_IN1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_OUT1">
    <property name="geometry">
     <rect>
      <x>919</x>
      <y>20</y>
      <width>471</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_Units">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>190</y>
      <width>651</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_POTENTIOMETER1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>660</y>
      <width>320</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_POTENTIOMETER1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>640</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>POTENTIOMETER</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_CLOCK">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>320</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_CLOCK">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CLOCK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
